chain/src/chain.rs

```diff
@@ -60,12 +63,7 @@ pub struct BlockInsertionResult {
 
 impl<CI: ChainIndex + 'static> ChainService<CI> {
     pub fn new(shared: Shared<CI>, notify: NotifyController) -> ChainService<CI> {
-        let block_verifier = BlockVerifier::new(shared.clone());
-        ChainService {
-            shared,
-            notify,
-            block_verifier,
-        }
+        ChainService { shared, notify }
```

Comment:
Why PR #140 is reverted here?

Reply:
the `consensus` is Arc<> now, `clone()` is not a problem. PR#140 should not be approved.